### PR TITLE
added error 500 page plus index ref

### PR DESCRIPTION
**Pull request by DomBaker**
Added error page and reference in index page.

### CODE REVIEW

File: src/pages/500.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends 'layout.html' %}
+
+{% block title %}Error{% endblock %}
+
+{% block content -%}
+  <h1>The NHS website is currently experiencing technical difficulties.</h1>
+  <p>We are working to resolve these issues. Thank you for your patience.</a></p>
+  <p>If you urgently need medical help or advice but it's not a life-threatening situation, call 111 free from any phone to speak to an NHS adviser. The 24-hour NHS 111 service can give you healthcare advice or direct you to the local service that can help you best.</p>
+{%- endblock %}
```
Comment (tomdoughty) [inline src/pages/500.html]: Add blank endline to keep git happy

File: src/pages/500.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% extends 'layout.html' %}
+
+{% block title %}Error{% endblock %}
+
+{% block content -%}
+  <h1>The NHS website is currently experiencing technical difficulties.</h1>
+  <p>We are working to resolve these issues. Thank you for your patience.</a></p>
```
Comment (tomdoughty) [inline src/pages/500.html]: Remove stray `</a>`

**Review by tomdoughty — CHANGES_REQUESTED** [review]
Just a couple of bits to fix

File: src/pages/500.html (hunk 1 above)
Comment (DomBaker) [reply src/pages/500.html]: no worries

File: src/pages/500.html (hunk 2 above)
Comment (DomBaker) [reply src/pages/500.html]: Good old copy and paste!

**Review by tomdoughty — APPROVED** [review]
LGTM 👍

**Pull request merged**